browser/aframeComponents/wearable-skin.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Component for selecting your character's skin in game.
+ */
+
+import AFRAME from 'aframe';
+const THREE = window.THREE;
+import { changeUserSkin } from '../utils';
+
+export default AFRAME.registerComponent('wearable-skin', {
+  schema: {
+    default: ''
+  },
+
+  // init binds this, creates event listeners for click and grip that both trigger the href handler, and setups the highlight effect.
+  init: function () {
+    this.handler = this.handler.bind(this);
+    this.setupHighlight = this.setupHighlight.bind(this);
+    this.el.addEventListener('click', this.handler);
+    this.el.addEventListener('gripdown', this.handler);
+    this.setupHighlight();
+  },
+
+  /**
+   * Called when component is removed.
+   */
+  remove: function () {
+    this.el.removeEventListener('click', this.handler);
+    this.el.removeEventListener('gripdown', this.handler);
+  },
+
+  // This will be the handler to start wearing a skin.
+  handler: function () {
+    console.log('Hey');
+    console.log(`Selected ${this.el.id}`);
+    changeUserSkin(this.el.id);
+    let msg = new SpeechSynthesisUtterance(`Changed skin to ${this.el.id}`);
+    window.speechSynthesis.speak(msg);
+  },
+
+  // setupHighlight creates a transluscent blue glow that is 20% larger than the shape with the href in all directions
```

Comment:
I almost think we could abstract this into its own component and throw it on to both this element and the orbs; seems like it would make it a bit more DRY.